core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -91,6 +91,10 @@ class KafkaServer(
   threadNamePrefix: Option[String] = None,
   enableForwarding: Boolean = false
 ) extends KafkaBroker with Server {
+  @volatile private var _brokerState: BrokerState = BrokerState.NOT_RUNNING
```

Comment:
Can we move this alongside the other vars and move the `defs` to after the vars? The reason is that `vars` are both mutable and definition of ordering matters. It's good to have them all in one place. `defs` definition order does not matter.